athensresearch/athens
# rfct(blocks): use ts anchor, toggle, and debug popover

- Replace block anchor (bullet) with TS component
- Replace block toggle with TS component
- Replace block debug popover (tooltip) with TS component

To test:
1. expand/collapse a block with the toggle
2. navigate into a block page by clicking the block button
3. rearrange some blocks by drag+dropping the block anchor
4. focus the block toggle and anchor using the keyboard
5. view the debug popover
   - activate the reframe-10x overlay (e.g. cmd-t)
   - hover a block anchor for 1500ms
   - popover should appear, and adjacent popovers should quickly appear
   - popovers should disappear after some time of inactivity
